### ISSUE TITLE
Posts save keeps running

### ISSUE THREAD

**Issue by Webagent05** [10up/distributor#346]
Hi there,

1. After enabling ... when I save a post the "save" keeps running. (no custom post type.)
![distributor-save-hangs](https://user-images.githubusercontent.com/48355318/54084364-fb4a8680-432f-11e9-9a58-f4513d17ed9d.jpg)

2. When I disable distributor, ... the issue is gone.

So something from distributor is interering saving posts. (It aren't distributed posts, but posts added in the concenring install.)

Hope to hear from you soon.

P.s. Where can I suggest features? I have some good ideas, ... I think. ;)

Kind regards,

Webagent05

**Comment by jeffpaul** [comment]
@Webagent05 can you please confirm the WordPress and Distributor versions you're using as well as any other plugins/themes so we can check for compatibility issues?

As for suggesting features, you can add those as issues here in GitHub as well... thanks!

**Comment by pafro** [comment]
@Webagent05 

I'm getting this "Save" always running too.  From a previous/current issue of yours I can see that we're both using the Newspaper theme.

I see this in the chrome console when it keeps showing 'Saving':
Request:
```
/EDIT/wp-admin/post.php?post=1&action=edit&meta-box-loader=1&_wpnonce=8d3e475b61&_locale=user
```
403 Response:
```
{"code":"wp_die","message":"The link you followed has expired.<\/p><p><a href=\"\/EDIT\/wp-admin\/post.php?post=1&#038;action=edit\">Please try again.<\/a>","data":{"status":403},"additional_errors":[]}
```

If I disable the Distributor plugin that exact request then works OK, returns 302

Distributor Plugin V1.4.0
WP version: 5.1.0
Latest Theme Version: 9.6.0

**Comment by Webagent05** [comment]
No interactions anymore with the developers?

Would love to see this plugin work well.

**Comment by jeffpaul** [comment]
@Webagent05 @Webagent05 I'd like to try your setup with the Newspaper theme as well, but in searching there are many available.  Can you confirm the slug for that theme or any further details so I can ensure I'm testing with the same theme?

**Comment by Webagent05** [comment]
@jeffpaul it's the "tagdiv newspaper theme", just paste in google.

**Comment by jeffpaul** [comment]
@Webagent05 @pafro ok, I see that the [Newspaper theme](https://themeforest.net/item/newspaper/5489609) is a paid theme on ThemeForest.  In lieu of me paying to test this out, could you switch to a  [theme from WordPress.org](https://wordpress.org/themes/) (e.g., [Twenty Nineteen](https://wordpress.org/themes/twentynineteen/))?  Also, I checked in the Gutenberg repo and see a couple issues related to saving that may also be related ([7564](https://github.com/WordPress/gutenberg/issues/7564) and [14302](https://github.com/WordPress/gutenberg/issues/14302)), you may want to subscribe to notifications on those to see if resolutions there help resolve this for you as well.

**Comment by Webagent05** [comment]
@jeffpaul deleting the connection and adding the connection again solved it.

I think I was to generous with pulling 30 posts in 1 time. This caused the issue twice.

**Comment by drebbits** [comment]
@jeffpaul I can confirm this issue, and I just opened a [PR](https://github.com/10up/distributor/pull/351) describing what the issue was.

I was able to reproduce with the following setup:

Distributor Plugin v1.4.0 (freshly installed/activated, no connection yet)
WP version: 5.0 to 5.1.1
Theme: Twenty Nineteen
Has custom meta boxes: Yes

**Comment by jeffpaul** [comment]
@Webagent05 @pafro note that we just released [1.4.1](https://github.com/10up/distributor/releases/tag/1.4.1) that may resolve this issue for you.  Please let me know your results after updating to 1.4.1, thanks!

**Comment by Webagent05** [comment]
@jeffpaul seems to be solved with updating to 1.4.1

**Comment by jeffpaul** [comment]
Thanks for confirming @Webagent05, I'll close this issue.
Resolved by #351.
